nengo_loihi/loihi_api.py

```diff
@@ -99,6 +99,46 @@ def tracing_mag_int_frac(synapses):
     return mag_int, mag_frac
 
 
+def decay_int(x, decay, bits=12, offset=0, out=None):
+    """Decay integer values using a decay constant.
+
+    The decayed value is given by
+
+        sign(x) * floor(abs(x) * (2**bits - offset - decay) / 2**bits)
+    """
+    if out is None:
+        out = np.zeros_like(x)
+    r = (2**bits - offset - np.asarray(decay)).astype(np.int64)
+    np.right_shift(np.abs(x) * r, bits, out=out)
+    return np.sign(x) * out
+
+
+def decay_magnitude(decay, x0=2**21, bits=12, offset=0):
+    """Estimate the sum of the series of rounded integer decays of `x0`.
```

Comment:
Single backticks here will likely give a Sphinx warning. If you want it to render like code, use double backticks. Some people use italics (`*x0*`) for parameter arguments instead; we haven't standardized so whatever you prefer.

Reply:
done